### PR TITLE
fix: divide by zero

### PR DESCRIPTION
**Pull request by user2745**

### CODE REVIEW

**Comment by user2745** [comment]
This fix sets the demoninator caused by 'games' to be reset to 1.  The reason is due to a 'divide by zero' error caused when an athlete has not played any games yet.

File: GetMLBAthleteData.py [hunk 1]
```diff
@@ -12,8 +12,8 @@
 HEADER = {'Ocp-Apim-Subscription-Key': apiKey }
 SDIO_URL = 'https://api.sportsdata.io/v3/mlb/stats/json/PlayerSeasonStats/2022'
 
-# HOST = 'localhost'
-HOST = '139.99.74.201'
+HOST = 'localhost'
```
Comment (powerMovesDev) [inline GetMLBAthleteData.py]: Revert so this will work in production

File: GetMLBAthleteData.py [hunk 2]
```diff
@@ -43,7 +43,10 @@ def computePrice(athlete_data, lgweightedOnBase, sumPlateAppearances):
         # collateralizationMultiplier = 1000
         BattingRuns = (((athlete_data['PlateAppearances']) * (athlete_data['WeightedOnBasePercentage'] - lgweightedOnBase)) / 1.25)
         BaseRunningRuns = (athlete_data['StolenBases'] * 0.2)
-        FieldingRuns = ((athlete_data['Errors'] * (-10)) / (athlete_data['Games'] * 9 ))
+        games = (athlete_data['Games'] * 9 )
+        if games <= 0:
+                games = 1
+        FieldingRuns = ((athlete_data['Errors'] * (-10)) / games)
```
Comment (powerMovesDev) [inline GetMLBAthleteData.py]: If games == 0 then fielding runs should be 0 since no games have been played right? Why make it one?

**Review by powerMovesDev — CHANGES_REQUESTED** [review]

File: GetMLBAthleteData.py (hunk 2 above)
Comment (ketonkss4) [reply GetMLBAthleteData.py]:
```
   FieldingRung = 0
   }else{
    FieldingRuns = ((athlete_data['Errors'] * (-10)) / games)
   }
   ```

File: GetMLBAthleteData.py (hunk 1 above)
Comment (ketonkss4) [reply GetMLBAthleteData.py]: `db.athletex.io`

**Review by ketonkss4 — CHANGES_REQUESTED** [review]

File: GetMLBAthleteData.py (hunk 1 above)
Comment (user2745) [reply GetMLBAthleteData.py]: completed

File: GetMLBAthleteData.py (hunk 2 above)
Comment (user2745) [reply GetMLBAthleteData.py]: Completed

**Review by ketonkss4 — APPROVED** [review]

File: GetMLBAthleteData.py (hunk 1 above)
Comment (ketonkss4) [reply GetMLBAthleteData.py]: realized we actually only support 443,80 ports to that load balancer

**Pull request merged**